### ISSUE TITLE
CRLF line terminators in script don't work under Linux/mac OS

### ISSUE THREAD

**Issue by grubersjoe** [leodr/generate-emoji-list#5]
Hey. Unfortunately, the script does not work on Linux/mac OS, because it has CRLF line terminators and LF terminators are expected here.

```sh
$ ./node_modules/.bin/generate-emoji-list
env: node\r: No such file or directory
$ file ./node_modules/.bin/generate-emoji-list
./node_modules/.bin/generate-emoji-list: a /usr/bin/env node script text executable, ASCII text, with CRLF line terminators
```

It works with `npx`, though. Maybe it has some magic inside to correct line terminators?

I suppose it should work everywhere if you switch to LF terminators.

Thanks

**Comment by leodr** [comment]
Yeah, I worked on Windows at the time. I now work on a MacBook so it should be easy to reproduce this and fix the line terminators.

**Comment by leodr** [comment]
Fixed with b560a59c6631602c8ea7b89075f8b4dbce118a25, released in version 1.0.2.

Release note following shortly.
